motioneye.json

```diff
@@ -9,8 +9,8 @@
         "nat_forwards": "tcp(8765:8765)"
     },
     "pkgs": [
-        "python37",
-        "py37-pip",
+        "python27",
+        "py27-pip",
```

Comment:
The plugin PR installing pip 27 in `post_install` has just been merged to `master`: https://github.com/cilix-lab/iocage-plugin-motioneye/pull/3. Could we try remove the `py27-pip` here now and see if it works better? :)

Reply:
I just tried installing this plugin with just the `python27` package manually on my machine. Seems like it passed without any errors and I could reach the motioneye UI. So hope this is a working quickfix :crossed_fingers:

Reply:
@stephenreda can you please confirm the above ? Thank you